peep.c

```diff
@@ -1,3 +1,23 @@
+/*    peep.c
+ *
+ *    Copyright (C) 1991-2022 by Larry Wall and others
+ *
+ *    You may distribute under the terms of either the GNU General Public
+ *    License or the Artistic License, as specified in the README file.
+ *
+ */
+
+/*
+ * 'Programmers waste enormous amounts of time thinking about, or worrying
```

Comment:
Having a non-standard quote feels more awkward to me than having no quote, TBH

Reply:
Ah; fair. I can remove it and queue it up with the other recently-added files to find a real LotR quote if that's preferred.